src/core/include/mp-units/framework/unit.h

```diff
@@ -633,7 +633,7 @@ template<Unit From, Unit To>
 
 template<Unit U1, Unit U2>
 [[nodiscard]] consteval Unit auto common_unit(U1 u1, U2 u2)
-  requires(convertible(u1, u2))
+  requires(convertible(MP_UNITS_IS_CONST_EXPR_WORKAROUND(u1), MP_UNITS_IS_CONST_EXPR_WORKAROUND(u2)))
```

Comment:
```suggestion
  requires(convertible(U1{}, U2{}))
```

and move up before the function signature.

Reply:
done